src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4770,11 +4770,19 @@ private static bool CanReuseVariableDeclarator(CSharp.Syntax.VariableDeclaratorS
                                 currentTokenKind == SyntaxKind.MinusGreaterThanToken ||
                                 isNonEqualsBinaryToken)
                             {
-                                var missingIdentifier = CreateMissingIdentifierToken();
-                                missingIdentifier = this.AddError(missingIdentifier, offset, width, ErrorCode.ERR_IdentifierExpected);
+                                var isPossibleLocalFunctionToken =
+                                    currentTokenKind == SyntaxKind.OpenParenToken |
```

Comment:
Also, why not just remove `currentTokenKind == SyntaxKind.OpenParenToken` from the enclosing `if` statement instead of testing for it again?

---
In reply to: [76164331](https://github.com/dotnet/roslyn/pull/13263#discussion_r76164331) [](ancestors = 76164331)